src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2835,6 +2838,11 @@ void processContextElement
       responseP->contextElementResponseVector.push_back(cerP);
     }
   }
+  else if (apiVersion == "v2" && action == "APPEND" && !isUpdate)
+  {
+    responseP->contextElementResponseVector[0]->statusCode.fill(SccInvalidModification, "Entity already exists");
+    return;
+  }
```

Comment:
What about that new mode called *APPEND_STRICT* or something similar ... ?
I may be wrong, but, this looks like a typical use for that 'mode'.

At least, the old APPEND (v1) becomes an UPDATE if the entity exists.
While new mode (I understood) would flag an error on 'already existing'.

Might be I have misunderstood the while thing, but ... enlighten me, please :-)